Gems/PhysX/Code/Include/PhysX/Joint/Configuration/PhysXJointConfiguration.h

```diff
@@ -112,4 +112,14 @@ namespace PhysX
         JointGenericProperties m_genericProperties;
         JointLimitProperties m_limitProperties;
     };
+
+    struct PrismaticJointConfiguration : public AzPhysics::JointConfiguration
```

Comment:
suggestion/nit: Might be worth adding an API comment here?

Reply:
Done

Reply:
Thanks!